redirect/routes.php

```diff
@@ -811,13 +811,13 @@ public function getTargetOfSub($host) {
         //In case we actually want to go to a different target than the actual redirect
         switch ($siteType) {
             case 'm' :
-                // This is a moodle redirect like m.info1.tum.sexy
-                $moodle_id = $this->routes[$redirectUrl]['moodle_id'];
-                if (!isset($moodle_id)) {
+                // This is a moodle redirect like minfo1.tum.sexy
+                $moodle_id = $this->routes[$redirectUrl];
```

Comment:
moodle_id is a bit misleading here as the variable doesn't contain the moodle id itself but rather the route. Other than that this looks good to me :+1: